reference/6/Microsoft.PowerShell.Core/New-ModuleManifest.md

```diff
@@ -1118,6 +1118,12 @@ that is installed in the global assembly cache.
 To add or change files in the `$pshome\Modules` directory, start PowerShell with the **Run as
 administrator** option.
 
+> [!NOTE]
+> Beginning in PowerShell 6.1, PowerShell attempts to load all the DLL files listed in **FileList**
```

Comment:
Shouldn't this be `6.2` given the related thread on the Issue?

Reply:
The change occurred in 6.1.